src/test/resources/features/ListDatasets.feature

```diff
@@ -9,6 +9,7 @@ Feature: List Datasets order by Title
     And I login as "user" with password "password"
     When I list the dataset
     Then The response code is 200
+    And User "owner" owns 1 dataset
```

Comment:
Removed "And User "owner" owns x dataset" from the scenarios, and added one scenario that list without authentication.